src/oop/special_class_method/hard/q5/QuestionText.md

```diff
@@ -0,0 +1,36 @@
+# oop-special_class_method-hard-q5
+
+### Road Class:
+
+Your `Road` class must have the following items:
```

Comment:
Please add the question context back to the question.

Reply:
For example, for this question the document had " You are writing a Java program that describes different types of roads and their attributes. ... ". You can use this first part and then stay something similar to "Using the requirements below, write a Java class called x: